spec/locale-sensitive-functions.html

```diff
@@ -75,7 +75,7 @@ <h1>
         </h1>
         <dl class="header">
           <dt>description</dt>
-          <dd>It interprets _S_ as a sequence of UTF-16 encoded code points, as described in <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and returns the result of implementation- and locale-dependent (ILD) transformation into _targetCase_ as a new String value.</dd>
+          <dd>It interprets _S_ as a sequence of UTF-16 encoded code points, as described in es2023, <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and returns the result of implementation- and locale-dependent (ILD) transformation into _targetCase_ as a new String value.</dd>
```

Comment:
```suggestion
          <dd>It interprets _S_ as a sequence of UTF-16 encoded code points, as described in ES2023, <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and returns the result of implementation- and locale-dependent (ILD) transformation into _targetCase_ as a new String value.</dd>
```
should ES2023 be linked to?

also maybe instead of adding the year, which would need to be updated every year, it could just say ECMAScript, and link to the living standard?

Reply:
The convention is described at https://tc39.es/ecma402/#normative-references :
> Throughout this document, the phrase “es2023, <var>x</var>” (where x is a sequence of numbers separated by periods) may be used as shorthand for "ECMAScript 2023 Language Specification (ECMA-262 14<sup>th</sup> Edition, sub clause <var>x</var>)".

It could definitely be changed, but doing so would be out of scope for this PR (which brings existing ECMA-262 references into alignment with it).

Reply:
oh ew, ok, fair enough